tests/test_google_sheets_datatypes.py

```diff
@@ -250,6 +245,10 @@ def test_run(self):
 
 
                             # verify the non-standard value has fallen back to a string type
+                            if column == 'Boolean' and value  in (-1, 1, 0): # special integer values falls back to boolean
```

Comment:
How many functions have we changed of singer Transform? IF only 1 or 2 functions we changed then can we override those functions instead of taking a complete transform here?

Reply:
Remove the file and overwritten the function instead